plutus-conformance/executables/haskell-implementation/Spec.hs

```diff
@@ -0,0 +1,8 @@
+{- | An executable for Haskell implementation CLI testing/usage. -}
```

Comment:
Should this be here? It doesn't do anything.

Reply:
Not right now but I like having the skeleton to remind our plans.